app/models/mirror.rb

```diff
@@ -0,0 +1,8 @@
+class Mirror < ActiveRecord::Base
```

Comment:
I don't know what do you think, but I'd prefer to rename them to `ImageMirror` and `ImageRegistry`. I don't have a strong opinion though, while `Registry` sounds good, `Mirror` sounds way too generic. Maybe `Registry` and `RegistryMirror`?